db/seeds/fill.rb

```diff
@@ -37,31 +39,21 @@
   end
 end
 
-Rails.logger.debug 'Seeding requests..'
-requests = FactoryBot.build_list(:request, request_count) do |request|
-  request.class.skip_callback(:create, :after, :broadcast_request, raise: false)
-  request.save!
-end
-
-Rails.logger.debug 'Seeding contributors..'
-contributors = FactoryBot.create_list(:contributor, contributors_count)
-
 FactoryBot.modify do
   factory :message do
-    created_at { message_time }
-    updated_at { message_time }
```

Comment:
I think I understand what you mean by:

> to have a valid order of timestamps on requests and replies

but I don't think I agree that this achieves that. My understanding would be that this would simply change the `created_at` and `updated_at` attrs from 14.days.ago to Time.current. How does that achieve a "valid order"? What am I missing?

I think what might be better is to use something like:

```rb
Faker::Time.between_dates(from:  Date.today - 14, to: Date.today, period: :all)
```

As I'm writing this, I realize that maybe you are trying to address the issue that the request itself was not created 14 days ago, so how could the messages come in/or go out before the request?

If that's the case, I still think it would be an improvement to set the `created_at` of the request to some time in the past and then set the replies to come in sometime between that date and today.

Reply:
The previous code just created a bunch of requests. After all requests have been created the messages are created on a batch. So if you look at conversations view you will find all requests at the top only then a bunch of contributor messages. To fix this you have to create a requests, send out the messages and make the contributor respond to them before sending the next request. Only then the order of message outbound/inbound is correct.